composer.json

```diff
@@ -5,7 +5,8 @@
     "license": "MIT",
     "require": {
         "php": ">=5.5.0",
-        "symfony/expression-language": "^3.0|^4.0"
+        "symfony/expression-language": "^3.0|^4.0",
+        "ruafozy/mersenne-twister": "^1.3"
```

Comment:
Hello, thanks for your PR. I was wondering why did you use "ruafozy/mersenne-twister" and then I read that mt_srand() is a  function that lead to side-effects as it initialize a seed for all the script execution, so this is not a proper solution.
However, I'm not excited of adding ruafozy/mersenne-twister as a dependency, as it's seems not to be a  library that is wide spread, the code seems to be aging and the last release is about 5 years old. I'm seeking for some other alternatives that could meet the same need, maybe you have some ideas ?

Reply:
There are several other contenders, that I haven't tested: https://phppackages.org/s/twister. I figured it was best to have a working implementation first, and then evaluate options. The age argument is I think understandable since it's an implementation of a known algorithm, there is not much to update in the lib until deprecation warnings start to show up.

We might explore a lib like https://github.com/http5/random, but it does not seem to be popular or updated frequently. Just more extensive.